taxi/views.py

```diff
@@ -34,8 +36,23 @@
     model = Manufacturer
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
+    queryset = Manufacturer.objects.all()
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
+        name = self.request.GET.get("name", "")
+        context["search_form"] = DriverSearchForm(
+            initial={"name": name}
+        )
```

Comment:
The `search_form` is initialized with `DriverSearchForm`, but it should be `ManufacturerSearchForm` to match the context of the `ManufacturerListView`.